modules/ROOT/pages/download/nb25/index.adoc

```diff
@@ -0,0 +1,151 @@
+* link:{url-download}netbeans-installers/{netbeans-version}/Apache-NetBeans-{netbeans-version}.pkg[Apache-NetBeans-{netbeans-version}.pkg] (link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/Apache-NetBeans-{netbeans-version}.pkg.sha512[SHA-512],
+link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/Apache-NetBeans-{netbeans-version}.pkg.asc[PGP ASC])
+* link:{url-download}netbeans-installers/{netbeans-version}/apache-netbeans_{netbeans-version}-1_all.deb[apache-netbeans_{netbeans-version}-1_all.deb] (link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans_{netbeans-version}-1_all.deb.sha512[SHA-512],
+link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans_{netbeans-version}-1_all.deb.asc[PGP ASC])
+* link:{url-download}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm[apache-netbeans-{netbeans-version}-0.noarch.rpm] (link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm.sha512[SHA-512],
+link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm.asc[PGP ASC])
+* link:https://snapcraft.io/netbeans[Linux snap package]
+
+*Source:*
+
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip[netbeans-{netbeans-version}-source.zip] 
+(link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.sha512[SHA-512], 
+link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.asc[PGP ASC])
+
+Officially, it is important that you link:https://www.apache.org/dyn/closer.lua#verify[verify the integrity]
+of the downloaded files using the PGP signatures (.asc file) or a hash (.sha512 files).
+The PGP keys used to sign this release are available link:https://downloads.apache.org/netbeans/KEYS[here].
+
+*Release Notes:*
+
+* link:https://github.com/apache/netbeans/releases/tag/{netbeans-version}[Github Link]
+
+== Community Installers
+
+- link:https://www.codelerity.com/netbeans/[Codelerity / Gj IT packages] - Windows, macOS and
+Linux (.deb / .rpm / .AppImage) built with
+link:https://github.com/apache/netbeans-nbpackage/[NBPackage]. Most
+include a local JDK runtime for the IDE to run on, for a self-contained out-of-the-box
+experience.
+
+IMPORTANT: Individual NetBeans committers may provide additional binary packages as a convenience.
+While built using the Apache NetBeans release, they are not releases of the Apache Software
+Foundation. They may include other contents (eg. JDK) under additional license terms.
+
+== Deployment Platforms
+
+Apache NetBeans {netbeans-version} supports running on JDK 24, 21 or 17. For the best experience we recommend using link:https://javaalmanac.io/jdk/24/[JDK 24].
```

Comment:
The support list should include JDK 23, and the recommended should be JDK 23 at least until 24 is released.

Reply:
when I updated the page two weeks ago i thought: 1) JDK release would be the first Tuesday, but it turns out to be the 18th and 2) we would have another release.
`Apache NetBeans {netbeans-version} supports running on JDK 24, 23, 21 or 17. For the best experience we recommend using link:https://javaalmanac.io/jdk/24/[JDK 24] once released.`

would this be better? Whatever we put on there shouldn't be changed IMO. Either we recommend 23 or 24 or nothing for this release.

Reply:
I think it might be better not to recommend a specific release this time around. But, a comment about making sure you're on the latest update with link(s) to where to check instead?

Reply:
the latest-update point is mentioned via the added TIP a few lines later. I will remove the second sentence.

Reply:
done

Reply:
Ah, yes, sorry missed that new TIP.  I think it's fine now.

In the past we've never treated unreleased JDKs at release time as supported, never mind recommended.  Any recommendation should be based on what has had most user testing during release candidate phase.  So I think recommending any specific JDK brings in wider scheduling conversations, and what you have now is a good compromise until that's resolved.